OracleWebLogic/samples/12212-msiserver/container-scripts/launcher.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/bash
```

Comment:
Yes, sorry. I meant handlers for `SIGKILL` and `SIGTERM` (which are what `docker stop` calls).